src/lang.rs

```diff
@@ -93,6 +93,7 @@ pub enum InlineOperator {
         pattern: Keyword,
         fields: Vec<String>,
         input_column: Option<Expr>,
+        parse_options: Vec<ParseOption>,
```

Comment:
lets just have this be a boolean `no_drop` -- I appreciate the forward thinking but might as well just keep it simple for now. It will simplify the downstream code as well